site/source/docs/porting/pthreads.rst

```diff
@@ -4,6 +4,8 @@
 Pthreads support
 ==============================
 
+Browsers are currently shipping SharedArrayBuffer gated behind Cross Origin Opener Policy (COOP) and Cross Origin Embedder Policy (COEP) headers. It is a possibility that user's pthreads code will not work in deployed environment.For more information click `this <https://web.dev/coop-coep>`_
```

Comment:
Please expand on the second sentence - it's not really a possibility, they will definitely not work across browsers unless the headers are correctly set.

Might be worth wrapping this into a note or a warning or something like that too.